app/shared/footer/FooterContent.js

```diff
@@ -56,6 +59,9 @@ function FooterContent({ t }) {
                 {feedbackLink}
               </p>
             </Col>
+            <Col lg={3} md={3}>
+              <p className="npm-version">v0.3.0</p>
```

Comment:
use more detail className. `app-varaamo-version` instead of generic one like `npm-version`